state_city.js

```diff
@@ -0,0 +1,43 @@
+
+const data = { 
```

Comment:
Give constant name in capital like `DATA`. Intead of `DATA` give some meaningful name like `COUNTRY_STATE_CITY_DD`